managed/src/test/java/com/yugabyte/yw/controllers/InstanceTypeControllerTest.java

```diff
@@ -101,16 +101,19 @@ private JsonNode doDeleteInstanceTypeAndVerify(UUID providerUUID, String instanc
     return Json.parse(contentAsString(result));
   }
 
-  private Map<String, InstanceType> setUpValidInstanceTypes(int numInstanceTypes) {
+  private Map<String, InstanceType> setUpValidInstanceTypes(int numInstanceTypes,
+                                                            String instanceTypesCode) {
     Map<String, InstanceType> instanceTypes = new HashMap<>(numInstanceTypes);
     for (int i = 0; i < numInstanceTypes; ++i) {
       InstanceType.VolumeDetails volDetails = new InstanceType.VolumeDetails();
       volDetails.volumeSizeGB = 100;
       volDetails.volumeType = InstanceType.VolumeType.EBS;
       InstanceTypeDetails instanceDetails = new InstanceTypeDetails();
       instanceDetails.volumeDetailsList.add(volDetails);
+      instanceDetails.volumeDetailsList.add(volDetails);
```

Comment:
Not obvious for me why we need to add it twice.

Reply:
@SergeyPotachev  Added it twice on purpose to check the volume list size for instance types not starting with "c5./c4."